packages/manager/apps/pci-workflow/src/pages/onboarding/OnBoarding.page.tsx

```diff
@@ -30,9 +26,12 @@ export default function OnBoardingPage() {
   const { data: project } = useProject();
   const urlProject = useProjectUrl('public-cloud');
   const navigate = useNavigate();
+  const projectUrl = useProjectUrl('public-cloud');
```

Comment:
You have a similar line at line 27, maybe use this similar variable ?

Reply:
Fixed